src/UI/Implementation/Component/Input/Field/Duration.php

```diff
@@ -7,8 +7,7 @@
 use ILIAS\UI\Component as C;
 use ILIAS\Data\Factory as DataFactory;
 use ILIAS\Data\DateFormat as DateFormat;
-use ILIAS\Refinery\Transformation\Factory as TransformationFactory;
-use ILIAS\Refinery\Validation\Factory as ValidationFactory;
+use ILIAS\Refinery as Refinary;
```

Comment:
```suggestion
use ILIAS\Refinery as Refinery;
```

The same problem repeats down below.